clarifai_model_utils/llm_eval/evaluator/harness_eval/doc/new_template.md

```diff
@@ -0,0 +1,174 @@
+There are 2 ways to add new task:
+
+# 1. Create task folder:
+
+Create new folder in [template](../template/) named it as the task name. The structure follows this [doc](https://github.com/EleutherAI/lm-evaluation-harness/blob/main/docs/task_guide.md)
+
+Suppose you want to use single metric says `f1` and you have your own implementation. So you will do:
+
+- Create folder named `f1` in `template`
+- Add files:
+
+  * f1.yaml: contains TaskConfig
+  ```yaml
+  group:
+  - f1
+  task: f1
```

Comment:
To clarify, do these instances of `f1` have anything to do with the method `f1`? Or this could be an arbitrary name? It can be confusing when using the same names as the method.